nflow-jetty/pom.xml

```diff
@@ -62,15 +62,15 @@
     </dependency>
     <dependency>
       <groupId>io.swagger.core.v3</groupId>
-      <artifactId>swagger-jaxrs2</artifactId>
+      <artifactId>swagger-jaxrs2-jakarta</artifactId>
```

Comment:
are the versions of the new deps defined in root pom? if not, they should be added.